tools/oesign/oesign.c

```diff
@@ -671,3 +765,64 @@ int oedigest(const char* enclave, const char* conffile, const char* digest_file)
 done:
     return ret;
 }
+
+char hexchar2int(char ch)
+{
+    if (ch >= '0' && ch <= '9')
+        return ch - '0';
+    if (ch >= 'a' && ch <= 'f')
+        return 10 + ch - 'a';
+    if (ch >= 'A' && ch <= 'F')
+        return 10 + ch - 'A';
+    return 0;
+}
+
+unsigned char hexpair2char(char a, char b)
+{
+    return (unsigned char)((hexchar2int(a) << 4) | hexchar2int(b));
+}
+
+int uuid_from_string(str_t* str, uint8_t* uuid, size_t expected_size)
+{
+    int rc = -1;
+    size_t index = 0;
+    size_t size = 0;
+    char* id_copy;
+    char value = 0;
+    bool firstDigit = true;
```

Comment:
Nit: use snake_casing for variable names per OE developer guidelines (i.e. `first_digit`)

Reply:
Updated